pkg/storage/sqlstorage/driver.go

```diff
@@ -143,41 +144,39 @@ func (d *Driver) GetConfiguration(ctx context.Context, key string) (string, erro
 	return value, nil
 }
 
-func (d *Driver) Initialize(ctx context.Context) (err error) {
+func (d *Driver) Initialize(ctx context.Context) error {
 	sharedlogging.GetLogger(ctx).Debugf("Initialize driver %s", d.name)
 
-	if err = d.db.Initialize(ctx); err != nil {
-		return
+	if err := d.db.Initialize(ctx); err != nil {
+		return fmt.Errorf("db.Initialize: %w", err)
```

Comment:
Prefer use errors.Wrap() as it allow to use errors.Unwrap() and errors.Is().

Use fmt.Errorf() shadow inner errors.